app/models/application_record.rb

```diff
@@ -1,3 +1,5 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
```

Comment:
@JackyT0n can you please change this to `false`?
```suggestion
  self.abstract_class = false
```